node/server.js

```diff
@@ -9742,25 +9742,27 @@ function init_io() {
 			);
 		});
 		socket.on("use", function (data) {
-			var player = players[socket.id];
+			const player = players[socket.id];
 			if (!player) {
 				return;
 			}
 			if (data.item == "hp" || data.item == "mp") {
-				if (player.last.potion && mssince(player.last.potion) < 0) {
-					return fail_response("not_ready");
+				if (player.last.potion) {
+					const ms = -mssince(player.last.potion);
+					if (ms > 0) {
+						socket.emit("eval", { code: `pot_timeout(${ms})` });
```

Comment:
Based on your gif, I'd say calling this function makes the game consider you've just used the associated skill and sets the timeout to `ms` which will throw of anything percentage based.

Reply:
Also, I would like us to remove `eval`s instead of adding more, when possible. I think it'll be better long term